hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/execution/HederaEvmTxProcessor.java

```diff
@@ -136,6 +138,7 @@ public HederaEvmTransactionProcessingResult execute(
                 .initialGas(gasAvailable)
                 .originator(senderEvmAddress)
                 .gasPrice(Wei.of(gasPrice))
+                .blobGasPrice((Wei.ONE))
```

Comment:
Will we ever use this value?

Reply:
It can be used if someone executes the `BLOBBASEFEE` opcode.  There won't be any blob transactions, but anyone can write a contract that uses the opcode.

It's set to `Wei.ZERO` by the `MessageFrame.Builder` constructor but our HIP says it should have the value `ONE`.